bundle/manifests/clusterlogging.clusterserviceversion.yaml

```diff
@@ -414,6 +415,8 @@ spec:
                   value: quay.io/openshift-logging/fluentd:1.14.6
                 - name: RELATED_IMAGE_LOG_FILE_METRIC_EXPORTER
                   value: quay.io/openshift-logging/log-file-metric-exporter:latest
+                - name: RELATED_IMAGE_LOGGING_CONSOLE_PLUGIN
+                  value: quay.io/openshift-logging/logging-view-plugin:latest
```

Comment:
Do we have CI building this image that should be configured in CI for this repo?

Reply:
I believe so @periklis ? The upstream quay.io image is one I pushed manually for now, @jgbernalp is going to automate it.

Reply:
https://github.com/openshift/release/pull/30506